src/Session/commands.py

```diff
@@ -1,14 +1,48 @@
 from src.Constants.commands_messages import *
 from src.Constants.constants import *
+from src.Food.food import Food
 from src.Session.profile import Profile
 from src.Utils.utils import listInQuotes, twoPrintOutcomes
 
 class Commands:
     def __init__(self):
         self.profile = None
 
+    ''' Prompts user to add foods to whichever food storage they are on. If the user is not in an opened 
+        food storage, suggests the user opens an existing food storage or create a new one. '''
     def add_food(self):
-        pass
+        # Profile check
+        if not self.profile:
+            print(NO_LOADED_PROFILE_MESSAGE, \
+                SUGGESTED_ACTIONS_MESSAGE.format(listInQuotes([CREATE_PROFILE, LOAD])))
+            return
+        
+        # Check if food storage is open
+        if self.profile.getOpenFoodStorage() != None:
+            food_list = []
+            while True: 
+                food_name = input(ADD_FOOD_NAME_MESSAGE)
+                expiration_date = input(ADD_FOOD_EXPIRATION_DATE_MESSAGE)
```

Comment:
do you guys have any error handling for if they input an invalid date? like if I type in 'lol' is the script going to error out and break or prompt me to enter a valid date?

Reply:
ya know i actually forgot about that lol we do have error handling for other things, but i forgot to do a check here to make sure it's valid input.

Reply:
Just added a way to check!

Reply:
lgtm